python/lib/sift_client/_internal/low_level_wrappers/ingestion.py

```diff
@@ -0,0 +1,405 @@
+            print(
+                f"Builder: {self.sift_stream_builder.apikey}, {self.sift_stream_builder.uri}, {self.sift_stream_builder.enable_tls}"
+            )
+            sift_stream = await self.sift_stream_builder.build()
+
+            ingestion_config_id = await self.get_ingestion_config_id_from_client_key(client_key)
+            print(f"Built new stream for ingestion config {ingestion_config_id}")
+            self._new_ingestion_thread(ingestion_config_id, sift_stream)
+
+        for flow in flows:
+            flow.ingestion_config_id = ingestion_config_id
+
+        return ingestion_config_id
+
+    def ingest_flow(
+        self,
+        *,
+        flow: Flow,
+        timestamp: datetime,
+        channel_values: dict[str, Any],
+        organization_id: str | None = None,
+    ):
+        """
+        Ingest a flow.
+        """
+
+        if not flow.ingestion_config_id:
+            raise ValueError(
+                "Flow has no ingestion config id -- have you created an ingestion config for this flow?"
+            )
+        sift_stream, data_queue, ingestion_task = self.stream_cache.get(
+            flow.ingestion_config_id, (None, None, None)
+        )
+        if sift_stream is None:
+            raise ValueError(
+                f"Ingestion config {flow.ingestion_config_id} not found. Have you created an ingestion config for this flow?"
+            )
+        # proto_channel_values = []
+        rust_channel_values = []
+        # Iterate through all expected channels for flow and convert to ingestion types (missing channels use a special empty type)
+        for channel in flow.channels:
+            val = channel_values.get(channel.name)
+            if val is None:
+                raise ValueError(f"Channel {channel.name} has no value")
+            if channel.data_type == ChannelDataType.ENUM and isinstance(val, str):
+                for enum_type in channel.enum_types or []:
+                    if enum_type.name == val:
+                        val = enum_type.key
+                        break
+            elif channel.data_type == ChannelDataType.BIT_FIELD:
+                # TODO: Handle bit field values correctly
+                val = None
+            # proto_channel_values.append(ChannelDataType.to_ingestion_value(channel.data_type, val))
+            rust_channel_values.append(
+                ChannelDataType.to_rust_value(channel.data_type, channel.name, val)
+            )
+        req = IngestWithConfigDataStreamRequestPy(
+            ingestion_config_id=flow.ingestion_config_id,
+            run_id=flow.run_id,
+            flow=flow.name,
+            timestamp=to_rust_py_timestamp(timestamp),
+            channel_values=rust_channel_values,
+            end_stream_on_validation_error=False,
+            organization_id=organization_id
+            or "",  # This will be filled in by the server # TODO: Confirm w/ Ailin
```

Comment:
Yes, we should be able to leave this unspecified -- the org ID can be resolved via the API key on the backend.